package.json

```diff
@@ -55,35 +55,35 @@
   },
   "dependencies": {
     "cliff": "^0.1.10",
-    "dot-object": "^1.9.0",
+    "dot-object": "^2.1.4",
     "liftoff": "^3.1.0",
-    "lodash": "^4.17.15",
-    "open": "6.4.0",
-    "request": "^2.88.0",
-    "semver": "^6.3.0",
-    "v8flags": "^3.1.3",
-    "yargs": "^14.0.0"
+    "lodash": "^4.17.20",
+    "open": "7.2.1",
+    "request": "^2.88.2",
+    "semver": "^7.3.2",
+    "v8flags": "^3.2.0",
+    "yargs": "^15.4.1"
   },
   "devDependencies": {
     "@types/expect.js": "^0.3.29",
-    "@types/mocha": "^5.2.7",
-    "@types/node": "^12.7.8",
+    "@types/mocha": "^8.0.3",
+    "@types/node": "^14.10.2",
     "copy-webpack-plugin": "^5.1.1",
```

Comment:
I was able to fix some vulnerabilities and still make it pass in Azure by doing a minor upgrade to v5.1.2.

https://github.com/portalarchitects/skysync-design-system/pull/570/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R67

Reply:
I noticed that - can do the same.

Reply:
Thanks. Upgraded to 5.1.2 and that got rid of the last high security vulnerability.